### PR TITLE
NoEmptyStatementFixer - handle anonymous classes correctly

### PR DESCRIPTION
**Pull request by gharlan**

### CODE REVIEW

File: Symfony/CS/Tests/Fixer/Symfony/NoEmptyStatementFixerTest.php [hunk 1]
```diff
@@ -493,7 +493,7 @@ public function log($msg)
             array(
                 '<?php
                     use function Functional\map;
-                    $a = new class extends A implements B {
+                    $a = new class extends \A implements B {
```
Comment (GrahamCampbell) [inline Symfony/CS/Tests/Fixer/Symfony/NoEmptyStatementFixerTest.php]: Best to add a new test rather than editing one?

**Comment by SpacePossum** [comment]
:+1: 
however I like the suggestion (https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1881/files#r60477545) to have a dedicated test :)

File: Symfony/CS/Tests/Fixer/Symfony/NoEmptyStatementFixerTest.php (hunk 1 above)
Comment (gharlan) [inline Symfony/CS/Tests/Fixer/Symfony/NoEmptyStatementFixerTest.php]: done

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by keradus** [comment]
@gharlan 
please, could you rebase?

**Comment by gharlan** [comment]
done

**Comment by GrahamCampbell** [comment]
:ship:

**Comment by keradus** [comment]
Thank you @gharlan.

**Pull request merged**